app/javascript/articles/Article.jsx

```diff
@@ -94,34 +93,35 @@ export const Article = ({
           )}
           {article.published_at_int ? (
             <span className="time-ago-indicator">
-              {timeAgoIndicator.length > 0 ? `(${timeAgoIndicator})`: '' }
+              {timeAgoIndicator.length > 0 ? `(${timeAgoIndicator})` : ''}
             </span>
           ) : null}
         </a>
       </h4>
-
       <TagList tags={article.tag_list} />
-      {article.class_name !== 'User' && (
-        <CommentsCount
-          count={article.comments_count}
-          articlePath={article.path}
-          icon={commentsIcon}
-        />
-      )}
-      {article.class_name !== 'User' && (
-        <ReactionsCount article={article} icon={reactionsIcon} />
-      )}
-      {article.class_name === 'Article' && (
-        <ReadingTime
-          articlePath={article.path}
-          readingTime={article.reading_time}
+      <div className="reactions-bar">
```

Comment:
The old HTML had the reactions and comments in reverse order in markup. With flexbox, I needed them in the correct order.